packages/internal/src/__tests__/fixtures/nestedPages/web/src/Routes.js

```diff
@@ -0,0 +1,59 @@
+// In this file, all Page components from 'src/pages` are auto-imported. Nested
+// directories are supported, and should be uppercase. Each subdirectory will be
+// prepended onto the component name.
+//
+// Examples:
+//
+// 'src/pages/HomePage/HomePage.js'         -> HomePage
+// 'src/pages/Admin/BooksPage/BooksPage.js' -> AdminBooksPage
+
+import { Private, Route, Router, Set } from '@redwoodjs/router'
+
+import AdminLayout from 'src/layouts/AdminLayout/AdminLayout'
+import MainLayout from 'src/layouts/MainLayout/MainLayout'
+
```

Comment:
```suggestion

import { someRandomExport } from 'src/pages/LoginPage'
```

Ideally our babel plugin would change that line to `import LoginPage, { someRandomExport } from 'src/pages/LoginPage'`, right? No point in adding an async loader for LoginPage if there already is a regular import for some other export in the file.

Reply:
Nah, that's assuming too much I think. If the user really cared about codesplitting, surely you don't put it in the same page file no? 

Simple is always better - and IMHO it really isn't worth handling a case like this.

Reply:
(you also need the async loader, because otherwise it won't know what LoginPage is)

Reply:
> (you also need the async loader, because otherwise it won't know what LoginPage is)

If the import was
```
import LoginPage, { someRandomExport } from 'src/pages/LoginPage'
```
Why would it also need the async loader?

> IMHO it really isn't worth handling a case like this.

Sure 👍  Let's leave it like it is until someone actually reports this as a problem.

Reply:
If they import like that, it'll be handled correctly (see last test case)